README.md

```diff
@@ -1,12 +1,17 @@
 # News Site
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The assignment was to create a responsive news site that contains a header, big news section and a grid of cards with other news. On bigger desktop sized screens, there should be four cards in a row, two in a row on tablets and on mobile, there should just be one card on each row.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+This is my first individual project and I am proud of what I managed to do and learn in the short amount of time.
+
+Things I struggled with was how to crop the images to the same aspect ratio without them being stretched or squished to fit and still having images be responsive. I learned how to use “object fit: cover” and that percentages (%) refers to the parent and vh stands for viewport height and vw stands for viewport width. I also struggled with how I should name the different classes to make them clear for both me and others who looks at my code.
```

Comment:
You definitely succeed. The images fit perfectly and your code is well structured!